app/controllers/stripe_controller.rb

```diff
@@ -19,6 +22,9 @@ def success_callback
     update_user_account
 
     flash[:notice] = "Subscription successful! Enjoy Mapzy!"
+
+    track_event("New Subscription", { subscription: params[:sub] })
```

Comment:
Couldn't these two also be written as before_action / after_action around the `checkout_session` action?

Reply:
Yes, good point. I wasn't sure if that would still work if e.g. there's an error, but it looks like it should not fire in that case. Fixed here: b6571fd